robolectric/src/main/java/org/robolectric/internal/bytecode/InstrumentationConfiguration.java

```diff
@@ -152,6 +153,7 @@ public InstrumentationConfiguration build() {
           DoNotInstrument.class,
           Config.class,
           Transcript.class,
+          DependencyResolver.class,
```

Comment:
I noticed that also and think that it is broken. Seems there are some types loaded by the system classsloader that shouldn't be. I believe all Android types should be loaded by the instrumenting classloader.